fdb-relational-grpc/src/main/java/com/apple/foundationdb/relational/jdbc/TypeConversion.java

```diff
@@ -581,6 +581,8 @@ public static Continuation.Reason toReason(RpcContinuationReason reason) {
     @SuppressWarnings("unchecked")
     static com.apple.foundationdb.relational.jdbc.grpc.v1.Options.Builder toProtobuf(@Nonnull Options options) throws SQLException {
         final var builder = com.apple.foundationdb.relational.jdbc.grpc.v1.Options.newBuilder();
+        // Switched-on by default on JDBC driver until the option is deprecated and removed.
+        builder.setContinuationsContainCompiledStatements(true);
```

Comment:
Is the idea here that we need to set this in case the `Options` from a modern version are shipped to a older server that expects the field to be set?

Reply:
The idea, from this version, is that the option cannot be explicitly set to False. Setting it to TRUE here is indicative of what we are getting into starting now. Also, the check here is to ensure that we never set it to False going ahead, or to maintain backward compatibility. After this is rolled out, we need a PR to remove usage of this option in JDBC and mark it deprecated. Then, we remove the field itself!